src/styles/comp.scss

```diff
@@ -0,0 +1,73 @@
+.comp {
+  display: flex;
+  flex-direction: column;
+  margin-inline: $margin-inline;
+  font-family: $font-family;
+
+  &__name {
+    padding-top: 207px;
+    font-weight: 500;
+    font-size: 12px;
+    align-self: center;
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-size: 160px 134px;
+    background-position: 0 33px;
+    color: $text-color-strong;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    margin-top: 4px;
+    color: $text-color;
+  }
+
+  &__rate {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+
+    &--stars {
+      display: flex;
+
+      &__star {
+        width: 16px;
+        height: 16px;
+        background-image: url(../images/star.svg);
+        background-repeat: no-repeat;
+        background-position: center;
+        margin-right: 4px;
+      }
+    }
+
+    &--stars--4 .comp__rate--stars__star:nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
+
+    &--reviews {
+      color: $text-color-strong;
+      font-weight: 400;
+      font-size: 10px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 24px;
+
+    &--1 {
+      color: $text-color;
+      font-weight: 400;
+      font-size: 12px;
+    }
+
+    &--2 {
+      color: $text-color-strong;
+      font-weight: 700;
+      font-size: 16px;
+    }
```

Comment:
The classes `comp__price--1` and `comp__price--2` do not clearly convey their purpose. In BEM, modifiers should describe the state or style of the block or element. Consider using more descriptive names, such as `comp__price--label` and `comp__price--value`.